settings.gradle

```diff
@@ -6,6 +6,7 @@ include 'fetch-doi-commons'
 include 'doi-proxy'
 include 'doi-transformer'
 include 'create-publication-from-doi'
+include 'download-scopus-publications'
```

Comment:
Dette navn er litt misvisende. Terje hadde også en task for "oss" at vi skulle downloade scopus fulltekst-filene. Først trodde jeg det handler om dette.

Reply:
"scopus-email-downloader"? What do you think about that?

Reply:
MyLambdaFunction vil toppe alt uansett.

Nei "scopus-email-file-downloader" er bra.